tailwind.config.js

```diff
@@ -5,6 +5,8 @@ export default {
     "./src/**/*.{html,js}",
     "./src/components/**/*.{html,js,tsx}",
     "./src/components/*.{html,js,tsx}",
+    "./src/pages/*.{html,js,tsx}",
```

Comment:
nitpick: tutti questi glob puoi semplificarli in uno singolo `./src/**/*.tsx`. un glob piu specifico puo rendere leggerissimamente piu veloce il tempo di compilazione di tailwind, perche deve controllare un range meno ampio di file per trovare le classi utilizzate.